internal/store/ring.go

```diff
@@ -16,17 +17,40 @@ var ringSelect sq.SelectBuilder
 
 func init() {
 	ringSelect = sq.
-		Select("Ring.ID", "Name", "Priority", "SoakTime", "Image", "Version", "Provisioner", "State", "CreateAt", "DeleteAt", "ReleaseAt", "APISecurityLock", "LockAcquiredBy", "LockAcquiredAt").
+		Select("Ring.ID", "Name", "Priority", "SoakTime", "Image", "Version", "Provisioner", "ChangeRequestMetadataRaw", "State", "CreateAt", "DeleteAt", "ReleaseAt", "APISecurityLock", "LockAcquiredBy", "LockAcquiredAt").
 		From("Ring")
 }
 
 type rawRing struct {
 	*model.Ring
+	*RawRingMetadata
+}
+
+// RawRingMetadata is the raw byte metadata for a ring.
+type RawRingMetadata struct {
```

Comment:
Do we expect to have more fields under this struct? If not I would get rid of it and just do `json.Marshal` instead of `buildRawMetadata`.